src/store/modules/user.js

```diff
@@ -17,14 +18,21 @@ export default {
                 const response = await httpClient.post("/api/auth/admin/login", { username, password }, { showLoader: false })
 
                 if (response.data && response.data.success === true) {
+                    const token = response.data.result.token;
                     localStorage.removeItem("token")
                     localStorage.removeItem("user")
-                    localStorage.setItem('token', response.data.result.token);
+                    localStorage.setItem('token', token);
                     localStorage.setItem("user", JSON.stringify(response.data.result.user))
                     sessionStorage.setItem('isAuth', 'true');
                     httpClient.defaults.headers.common['Authorization'] = 'Bearer ' + response.data.result.token;
                     commit("updateLoggedinUser", response.data.result.user)
-                    router.push('/dashboard')
+                    const decodedToken = jwtDecode(token)
+                    const role = decodedToken['http://schemas.microsoft.com/ws/2008/06/identity/claims/role'];
+                    if (role == 'SuperAdmin' || role == 'Admin') {
+                        router.push('/dashboard')
+                    } else if (role == 'Checker') {
```

Comment:
please don't use strings like this. there is a constant in appdefault.js and use it
![image](https://github.com/naeimrezaeian/ACT-ADMIN/assets/6144486/1cdcc931-ecad-453f-bc7e-1adab7f49891)
also fix condition like this
if(role=='checker'){
goto('useexam')
else
goto('dashboard')
}